### PR TITLE
Use system accent color

### PR DESCRIPTION
**Pull request by lenuswalker**
Option to allow app accent color to use system accent color. This will address part of the feature request in issue #200.

Need to add some more logic if we want to follow the system accent color.

### CODE REVIEW

File: LenovoLegionToolkit.WPF/Controls/ColorCardControl.cs [hunk 1]
```diff
@@ -14,6 +15,8 @@ namespace LenovoLegionToolkit.WPF.Controls
 {
     public class ColorCardControl : UserControl
```
Comment (BartoszCichecki) [inline LenovoLegionToolkit.WPF/Controls/ColorCardControl.cs:16]: This control is also used for RGB keyboard controls so it's not a good idea to add the system accent color checkbox here.

File: LenovoLegionToolkit.WPF/Controls/ColorCardControl.cs [hunk 2]
```diff
@@ -196,6 +249,36 @@ private void ColorsPanel_MouseUp(object sender, MouseButtonEventArgs e)
             OnChanged?.Invoke(this, EventArgs.Empty);
         }
 
+        private void SystemAccentColor_Checked(object sender, RoutedEventArgs e)
+        {
+            Color systemAccentColor = ((SolidColorBrush)SystemParameters.WindowGlassBrush).Color;
```
Comment (BartoszCichecki) [inline LenovoLegionToolkit.WPF/Controls/ColorCardControl.cs:254]: Does this work on Windows 10 and Windows 11?

**Review by BartoszCichecki — CHANGES_REQUESTED** [review]
Because color card control is used for RGB keyboard settings I think it might be a good idea to put one more row on the settings page, for example below the current color picker. It can look like the Autorun setting, and once enabled it would set `IsEnabled` to false on the current color picker.

To consider it complete, we need also to listen for system settings so that LLT responds dynamically to color changes.

**Comment by BartoszCichecki** [comment]
And same comment as the other PR:

Thanks a lot for the contributions, next week I will have more time to look at them and I hope we can get them merged!

File: LenovoLegionToolkit.WPF/Controls/ColorCardControl.cs (hunk 2 above)
Comment (lenuswalker) [reply LenovoLegionToolkit.WPF/Controls/ColorCardControl.cs:254]: I'm using Windows 11 on all my devices. Don't have a Windows 10 to test.

File: LenovoLegionToolkit.WPF/Controls/ColorCardControl.cs (hunk 1 above)
Comment (lenuswalker) [reply LenovoLegionToolkit.WPF/Controls/ColorCardControl.cs:16]: > 

Interesting. I don't have a Lenovo machine. Does the Accent Color setting in the Color Card Control also set any colors for the Keyboard?

I'll try to separate it from the color card control and add something probably directly under the Theme settings for App Accent Color?

**Comment by lenuswalker** [comment]
Take a look at the last commit for the system accent color. Redid it to add the switch on the settings page and actively monitor changes to the system accent color.

**Comment by BartoszCichecki** [comment]
Hey, closing this one, because some other guy made a similar PR, but in his also the keyboard RGB could follow system accent.

**Comment by BartoszCichecki** [comment]
Ref: #207

**Comment by lenuswalker** [comment]
Yea I saw that PR. It seemed awesome.

**Pull request closed without merging**